### PR TITLE
[#175892280] Update max body size for logo update

### PR DESCRIPTION
**Pull request by infantesimone**
This PR update max body size to 50 mb for the functions:
- UploadOrganizationLogo
- UploadServiceLogo

### CODE REVIEW

File: UploadOrganizationLogo/index.ts [hunk 1]
```diff
@@ -30,6 +32,8 @@ winston.add(contextTransport);
 const app = express();
 secureExpressApp(app);
 
+app.use(bodyParser.json({ limit: "50mb" }));
```
Comment (gunzip) [inline UploadOrganizationLogo/index.ts:35]: is this a reasonable limit? (what's the expected max size? the current one?)

**Review by AleDore — COMMENTED** [review]
LGTM @gunzip ?

File: UploadOrganizationLogo/index.ts (hunk 1 above)
Comment (AleDore) [reply UploadOrganizationLogo/index.ts:35]: Navigating through [io-services-metadata](https://github.com/pagopa/io-services-metadata/tree/master/logos) it seems that the expected max size is 1 Mb. With a conservative approach we can define a limit up to 5-10 MB.

Comment (gunzip) [reply UploadOrganizationLogo/index.ts:35]: yes, can we update to 5mb?

Comment (infantesimone) [reply UploadOrganizationLogo/index.ts:35]: ok 👍

**Review by AleDore — APPROVED** [review]
LGTM

**Comment by balanza** [comment]
We are experiencing some troubles with PR's automatic check. We close this PR and we open a new one, let's see if this solves the issue.

**Pull request closed without merging**